src/utils/getNumberOfFileChanges.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from "vscode";
+import getFullBlame from "./getFullBlame";
+
+async function getNumberOfFileChanges(gitAPI:any, lineNumber:number) {
```

Comment:
This is gonna be used on hover, not necessarily on the _active_ file.
Please receive the param

Reply:
Ok